python/tvm/autotvm/task/relay_integration.py

```diff
@@ -31,23 +31,28 @@
 logger = logging.getLogger('autotvm')
 
 
-# TODO(moreau89) find a more elegant way to build for VTAs
-def _build(func,
+# TODO(moreau89) find a more elegant way to lower for VTAs
+def _lower(func,
```

Comment:
Do we need test cases for extract_from_program? I didn't find any existing one.

Reply:
We do have unit tests for extract_from_program.
My just merged PR is related to this one: https://github.com/dmlc/tvm/pull/4173

Reply:
> Do we need test cases for extract_from_program? I didn't find any existing one.

I do have my own test case for justification and performance evaluation.

> We do have unit tests for extract_from_program.
> My just merged PR is related to this one: #4173

As this PR has been merged, maybe I should just skip mine and directly use this one?

Reply:
I think it's fine. After resolving the conflict your changes should be covered by existing unit tests (for functionality).